app/controllers/api/account.php

```diff
@@ -1500,25 +1500,27 @@
         $response->dynamic($user, Response::MODEL_USER);
     });
 
-App::delete('/v1/account')
-    ->desc('Delete Account')
+App::patch('/v1/account')
+    ->desc('Update Status (Block)')
     ->groups(['api', 'account'])
-    ->label('event', 'account.delete')
+    ->label('event', 'account.updateStatus')
     ->label('scope', 'account')
     ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_JWT])
     ->label('sdk.namespace', 'account')
-    ->label('sdk.method', 'delete')
-    ->label('sdk.description', '/docs/references/account/delete.md')
-    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
-    ->label('sdk.response.model', Response::MODEL_NONE)
+    ->label('sdk.method', 'block')
+    ->label('sdk.description', '/docs/references/account/block.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_USER)
```

Comment:
```suggestion
    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
    ->label('sdk.response.model', Response::MODEL_NONE)
```
Lets stay consistent with the pattern we follow for delete endpoints

Reply:
As mentioned above, this is `PATCH`. Do we rename to PATCH, keep the response model, or keep patch and do no response?

Reply:
I see. Okay 👍  Thanks for confirming. We can return the user object